docs/framework/additional-apis/adodb.eventreasonenum.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: EventReason Enumeration (adodb)
+description: Get information about the EventReason enumeration in .NET. This interface is in the adodb namespace and the adodb assembly (in the adodb.dll library).
```

Comment:
> the ... enumeration ... This interface

Is it an enum or an interface?

Reply:
Yes, that was a copy/paste error. Good catch.